src/draco/io/stl_decoder.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 The Draco Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "draco/io/stl_decoder.h"
+
+#include "draco/core/macros.h"
+#include "draco/core/status.h"
+#include "draco/core/status_or.h"
+#include "draco/io/file_utils.h"
+#include "draco/mesh/triangle_soup_mesh_builder.h"
+
+namespace draco {
+
+StatusOr<std::unique_ptr<Mesh>> StlDecoder::DecodeFromFile(
+    const std::string &file_name) {
+  std::vector<char> solid_keyword = {'s', 'o', 'l', 'i', 'd', ' '};
```

Comment:
nit: can this be std::array or some other static storage data structure. No need to use dynamic vector when the content doesn't change.

Reply:
Done